pkg/arvo/sys/vane/gall.hoon

```diff
@@ -610,12 +610,16 @@
         [%a %done *]
       =^  remote-request  outstanding.agents.state
         ?~  t.t.t.wire
+          =/  full-wire  sys+wire
           =/  stand
-            %+  ~(gut by outstanding.agents.state)  [sys+wire hen]
-            *(qeu remote-request)
-          ~|  [sys+wire=wire hen=hen stand=stand outs=outstanding.agents.state]
+            %+  ~(gut by outstanding.agents.state)  [full-wire hen]
+            ::  default is do nothing; should only hit if cleared queue
+            ::  in +load 3-to-4
+            ::
+            (~(put to *(qeu remote-request)) %leave)
```

Comment:
Why do we insert a %leave into the queue of outstanding requests if it was empty?  I get that we might have cleared the queue during migration, but I don't see how this would help.  I feel like I'm misunderstanding something here.

We could be in the middle of some other request that isn't a %leave, right?  ... in which case when we hear the ack on whatever that request was, we'll misinterpret it as an ack on this %leave that we never actually sent.

Reply:
Yeah, `%leave` isn't correct, but it's impossible to know for certain the right answer.  This feels conservative because it'll just give no acknowledgment at all.

It occurs to me this could screw up `%ping` because it's expecting a poke-ack, and if it doesn't get it, then it might hang.  Don't really have a good answer.  I could just fix it to `%poke`, which also feels not great, but it's less likely to cause a real problem I guess.